### PR TITLE
Event QR Code pages: Explanation about outdated page info in blue box 

### PR DESCRIPTION
**Pull request by dsarkar**
See https://github.com/corona-warn-app/cwa-website/issues/3484

Blue info-box added to:

1. https://www.coronawarn.app/de/eventregistration/
2.  https://www.coronawarn.app/en/eventregistration/
3.  https://www.coronawarn.app/en/event-qr-code-guide/
4. https://www.coronawarn.app/de/event-qr-code-guide/

**- DE info box**
image

**- EN info box**
image

---
Internal Tracking ID: [EXPOSUREAPP-15169](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-15169)

### CODE REVIEW

**Review by mtb77 — APPROVED** [review]

File: src/data/eventregistration.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+    "section-notice": {
+        "textblock": [
+            "<div class='update'>Since it is no longer possible to warn other users through the Corona-Warn-App regarding an increased risk of infection as of May 1, 2023, you can no longer use the QR codes created on this page or other tools to register for events and locations. As such, the information on this page is related to features of the app that are no longer available as of May 1, 2023. For more information, click here: <a href='../faq/results/#ramp_down' target='_blank'> What is the background for the end of the development of the CWA?</a></div>"
```
[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/eventregistration.json:4]: I think the English texts are not quite right, are they?

[the remark this answers is not in the document]
Comment (MikeMcC399) [reply src/data/eventregistration.json:4]: "created on other tools" would also be correct, so this suggestion is OK, but not necessary.

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Thanks for reviewing and commenting,

**Comment by Ein-Tim** [comment]
@dsarkar If you apply my suggested change to one of the texts, you should also apply it to the other occurrences.

**Pull request merged**